cypress/integration/cohorts.js

```diff
@@ -27,7 +27,7 @@ describe('Cohorts', () => {
 
         // save
         cy.get('[data-attr="save-cohort"]').click()
-        cy.get('[data-attr=success-toast]').should('exist')
+        cy.get('.Toastify__toast').contains('Cohort saved').should('exist')
```

Comment:
Should we keep the `data-attr` on the toasts? This could be used for instrumentation somewhere... and is programmatically better than relying on styling hints (css classes) when pinpointing content... 🤔